src/devices/custom_devices_diy.ts

```diff
@@ -148,6 +153,22 @@ const tzLocal = {
             };
         },
     } as Tz.Converter,
+    multi_zig_sw_switch_type: {
+        key: ['switch_type'],
+        convertGet: async (entity, key, meta) => {
+            await entity.read('genOnOffSwitchCfg', ['switchType']);
+        },
+        convertSet: async (entity, key, value, meta) => {
+            if (key == 'switch_type') {
```

Comment:
No need for this if statement, the key will always be `switch_type` because of `key: ['switch_type'],`